test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageExtractorTests.cs

```diff
@@ -5111,12 +5131,64 @@ public async Task InstallFromSourceAsync_WithPackageDownloader_LogsSourceOnNorma
             }
         }
 
+        // This test is likely to only catch bugs on Linux/Mac, but there's little harm in running the test
+        // on Windows as well.
+        [Fact]
+        public async Task InstallFromSourceAsync_NupkgWithNonNormalizedPaths_FixesPathsToExtract()
```

Comment:
Did you try this test before your change. I assume it failed on Linux or Mac or at least you're confident it would have failed? It definitely succeeds on Windows without this change as I tested that.